### ISSUE TITLE
[ReadMe/FAQ suggestion]

### ISSUE THREAD

**Issue by crazyman10123** [jabbink/PokemonGoBot#825]
Just a short suggestion that I noticed for the FAQ.

For JAR File inaccessible issue you might also need to check the properties of the file to make sure Windows/Linux isn't blocking it from being run. 

I know for sure on Windows 7 you can right click it and you'll see the Unblock button at the bottom of the first Window that pops up after you click properties. Not sure if Win10 is the same. Linux (or at least Ubuntu) should have an option to allow the file to be executed.

**Comment by smileytechguy** [comment]
Its suggested to just run java -jar Pokemongobot.jar

**Comment by crazyman10123** [comment]
@smileytechguy yes I know, but that still returns inaccessible sometimes if it's still being blocked. I've experienced it myself before.
